### PR TITLE
Adding autoplay to siteditor properties

### PR DESCRIPTION
**Pull request by julikroch**
#### What problem is this solving?

Adding autoplay properties so it can be changed for the sliders from VTEX site editor without code development.

#### How to test it?

https://julianslider--pivotree.myvtex.com/

#### Screenshots or example usage:

schema

### CODE REVIEW

**Comment by julikroch** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [ ]  There's nothing new to document 🤔
> [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> [ ]  I just updated 🎉🎉

**Comment by julikroch** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞



> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Comment by julikroch** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: messages/pt.json [hunk 1]
```diff
@@ -5,5 +5,8 @@
   "admin/editor.slider-layout.showPaginationDots": "Mostrar indicadores de paginação",
   "admin/editor.slider-layout.usePagination": "Usar paginação",
   "admin/editor.slider-layout.sliderFullWidth": "Largura completa",
-  "admin/editor.slider-layout.sliderFullWidthDescription": "Controla se os slides devem ou não ocupar toda a largura disponível, fazendo as setas aparecerem sobre eles."
+  "admin/editor.slider-layout.sliderFullWidthDescription": "Controla se os slides devem ou não ocupar toda a largura disponível, fazendo as setas aparecerem sobre eles.",
+  "admin/editor.slider-layout.autoplay": "Autoplay",
+  "admin/editor.slider-layout.stopOnHover": "Autoplay's Stop on hover",
+  "admin/editor.slider-layout.timeout": "Autoplay's Timeout"
```
Comment (camilavcoutinho) [inline messages/pt.json:11]:
```suggestion
  "admin/editor.slider-layout.autoplay": "Reproduzir automaticamente",
  "admin/editor.slider-layout.stopOnHover": "Parar reprodução automática ao passar o cursor",
  "admin/editor.slider-layout.timeout": "Duração da reprodução automática"
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Comment by julikroch** [comment]
@camilavcoutinho all your changes were approved. Thank you! :)

**Review by camilavcoutinho — APPROVED** [review]

**Pull request closed without merging**